### PR TITLE
Update for OSX 10.11 (El Capitan)

### PR DESCRIPTION
**Pull request by joscha**
This updates the pkg details for El Capitan (OSX 10.11)

### CODE REVIEW

File: Casks/haskell-platform.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
 cask :v1 => 'haskell-platform' do
-  version '7.10.2-a'
-  sha256 'dd1b64ecec95178044e12a08d9038f1e2156bbd51537da07b18832531b637672'
+  
+  if MacOS.release >= :el_capitan
```
Comment (vitorgalvao) [inline Casks/haskell-platform.rb]: Conditionals should go the other way around.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in d09e301a9d80d4281b66635cfb12fa9f40c4d1c8. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/haskell-platform.rb (hunk 1 above)
Comment (joscha) [inline Casks/haskell-platform.rb]: Sorry, a bit confused - I want to install `7.10.2-a1` on Capitan or higher, so `MacOX.release (assume 10.11) >= :el_capitan (10.11)` installs the correct version or am I missing something?

Comment (joscha) [inline Casks/haskell-platform.rb]: Took it from [here](https://github.com/caskroom/homebrew-cask/search?utf8=%E2%9C%93&q=%22if+MacOS.release+%3E%3D+%3Ael_capitan%22&type=Code)

**Comment by joscha** [comment]
:+1:

**Pull request closed without merging**